packages/client/src/contents/types/seoFiles.types.ts

```diff
@@ -0,0 +1,40 @@
+import type { Config } from '../../types/index.js';
+
+export type GetSEOFiles = (
+  query: GetSEOFilesQuery,
+  config?: Config,
+) => Promise<SEOFiles>;
+
+export enum SeoFileType {
+  None = 'None',
+  Sitemap = 'Sitemap',
+  Homepage = 'Homepage',
+  Pages = 'Pages',
+  Posts = 'Posts',
+  Products = 'Products',
+  Sets = 'Sets',
+  Other = 'Other',
+  CustomContentTypes = 'CustomContentTypes',
+  Categories = 'Categories',
+}
+
+export type GetSEOFilesQuery = {
+  // The name (also named as "slug" on legacy CMS)
+  name: string;
+  // The hostId
+  hostId: number;
+  // Number of the page to get, starting at 1. The default is 1.
+  page: number;
+  // Size of each page, as a number between 1 and 180. The default is 60.
+  pageSize: number;
+};
+
+export type SEOFiles = {
```

Comment:
This should be singular as it represents one single file.

```suggestion
export type SEOFile = {
```

Reply:
Resolved! ✅